roles/provision-keycloak-apb/tasks/provision-keycloak.yml

```diff
@@ -65,6 +65,8 @@
   k8s_v1_service:
     name: keycloak
     namespace: '{{ namespace }}'
+    annotations:
+      org.aerogear.metrics/plain_endpoint:  /auth/realms/master/metrics
```

Comment:
:+1: 

this only works w/ the custom JAR (SPI impl) included - perhaps worth a comment  ?

Reply:
Ok cool, will add that in now :+1: